glocaltokens/scanner.py

```
@@ -0,0 +1,146 @@
+from threading import Event
+import zeroconf
+import logging
+
+from .utils import network as net_utils
+from .utils import type as type_utils
+
+DISCOVER_TIMEOUT = 5
+GOOGLE_HOME_MODELS = [
+    "Google Home",
+    "Google Home Mini",
+    "Google Nest Mini",
+    "Lenovo Smart Clock"
+]
```

Can we please add a few words to the README to inform the user that currently, we support these Google Home models. 
Otherwise, some of Google Home device discovery can silently fail. For example, I have Google Nest Audio at home which will probably not be discovered as of now :)

---

Yeah, will add this, it's quite important. Also, this is kind of a pre-defined const, you could create your own models list. Anyway, I'm adding it to README so it's clear.

---

Added in ca9dd2d